<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>ajouter cours</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>ajout</string>
          </property>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>30</y>
             <width>47</width>
             <height>14</height>
            </rect>
           </property>
           <property name="text">
            <string>id cours</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>80</y>
             <width>81</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>nom matiere</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_10">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>130</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>nom proffeseur</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_11">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>180</y>
             <width>101</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>prenom proffeseur</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_12">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>220</y>
             <width>101</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>telephone proffeseur</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_13">
           <property name="geometry">
            <rect>
             <x>290</x>
             <y>50</y>
             <width>47</width>
             <height>14</height>
            </rect>
           </property>
           <property name="text">
            <string>prix</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_14">
           <property name="geometry">
            <rect>
             <x>290</x>
             <y>150</y>
             <width>47</width>
             <height>14</height>
            </rect>
           </property>
           <property name="text">
            <string>type</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="le_id_Cours">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>30</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
          <widget class="QLineEdit" name="le_nom_matiere">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>80</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
          <widget class="QLineEdit" name="le_nom_proffeseur">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>130</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
          <widget class="QLineEdit" name="le_prenom_proffeseur">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>180</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
          <widget class="QLineEdit" name="telephone_proffeseur">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>220</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
          <widget class="QLineEdit" name="le_prix">
           <property name="geometry">
            <rect>
             <x>340</x>
             <y>50</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
          <widget class="QLineEdit" name="le_type">
           <property name="geometry">
            <rect>
             <x>340</x>
             <y>150</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
          <widget class="QPushButton" name="pb_ajouter">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>220</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>ajouter</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_2">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>190</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Modifier</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget">
       <attribute name="title">
        <string>afficher cours</string>
       </attribute>
       <widget class="QTableView" name="tab_cours">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>521</width>
          <height>211</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Trier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>10</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Actualiser</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>supprimer cours</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>100</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>100</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Statistiques</string>
       </attribute>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>20</y>
          <width>256</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>30</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>actualiser</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
